src/openrct2/ride/Ride.cpp

```diff
@@ -1099,9 +1096,9 @@ void ride_remove_peeps(Ride* ride)
     }
 
     // Place all the peeps at exit
-    uint16_t spriteIndex;
-    Peep* peep;
-    FOR_ALL_PEEPS (spriteIndex, peep)
+    // TODO confirm if mechanics can get here!
```

Comment:
Why? Do you think this is a current issue? If so, might be worth mentioning the #

Reply:
I think thats a copy paste mistake will remove TODO